src/dlls/mscoree/coreclr/CMakeLists.txt

```diff
@@ -133,12 +128,6 @@ else()
     )
 endif(WIN32)
 
-if(CLR_CMAKE_PLATFORM_UNIX AND FEATURE_STANDALONE_GC)
-    list(APPEND CORECLR_LIBRARIES
-        gc_unix
-    )
-endif(CLR_CMAKE_PLATFORM_UNIX AND FEATURE_STANDALONE_GC)
```

Comment:
Nothing in this file is necessary unless we are linking a standalone GC into the runtime itself, which we don't need to do anymore because it can be loaded dynamically.